gulpfile.js

```diff
@@ -8,6 +8,12 @@ var testPath = "./test";
 var binPath = "./bin";
 var tsFiles = "/**/*.ts";
 
+var androidEmulatorName = "emulator";
+var iOSEmulatorName = "iPhone 6s Plus (9.3) [";
```

Comment:
yes, because there is a "iPhone 6s Plus (9.3) with Apple Watch" simulator as well so the "[" has to be there to specify that it's just the "iPhone 6s Plus (9.3)" emulator